Watchman.Discord/Areas/Initialization/Services/InitializationService.cs

```diff
@@ -38,13 +38,38 @@ public InitializationService(IQueryBus queryBus, ICommandBus commandBus, MuteRol
 
         public async Task InitServer(DiscordServerContext server)
         {
+            await AddDefaultCustomCommands();
             await MuteRoleInit(server);
             var lastInitDate = GetLastInitDate(server);
             await ReadServerMessagesHistory(server, lastInitDate);
             await _cyclicStatisticsGeneratorService.GenerateStatsForDaysBefore(server, lastInitDate);
             await NotifyDomainAboutInit(server);
         }
 
+        private async Task AddDefaultCustomCommands()
+        {
+            var customCommands = new List<AddCustomCommandsCommand>();
+            foreach (var serverId in new ulong[] { 636238466899902504, 597066406521208852 })
+            {
+                customCommands.AddRange(new List<AddCustomCommandsCommand>
+                {
+                    new AddCustomCommandsCommand("Watchman.Discord.Areas.Users.BotCommands.AddRoleCommand", @"-add\s*role\s*(?<Roles>[\w\s\""]*)", serverId),
+                    new AddCustomCommandsCommand("Watchman.Discord.Areas.Users.BotCommands.RemoveRoleCommand", @"-remove\s*role\s*(?<Roles>[\w\s\""]*)", serverId),
+                    new AddCustomCommandsCommand("Watchman.Discord.Areas.Administration.BotCommands.SetRoleCommand", @"-set\s*role\s*(?<Roles>[\w\s\""]*)\s*-*(?<Safe>safe)*\s*-*(?<Unsafe>unsafe)*", serverId),
```

Comment:
(?<Unsafe>unsafe)* 
czemu ta * na końcu?

Reply:
dlatego, że nie zawsze musi być grupa unsafe
jeśli jest już -safe to nie będzie -unsafe i na odwrót

Reply:
no to wtedy daje sie `?` na końcu - (możliwe wyniki to 1 albo 0)
`*` daje sie jak chcesz mieć dużo elementów, ale nie masz pewności że jakikolwiek będzie (możliwe wyniki to od 0 do nieskończoności)

Reply:
ale ja wrzuciłbym tam | 
`-((?<Safe>safe)|(?<Unsafe>unsafe))`

Reply:
żeby nie było opcji że ktoś wpisze `-safe -unsafe`